plugins/package-managers/conan/src/main/kotlin/Conan.kt

```diff
@@ -488,5 +488,5 @@ class Conan(
      * author name; otherwise, return an empty set.
      */
     private fun parseAuthors(node: JsonNode): Set<String> =
-        parseAuthorString(node["author"]?.textValue(), '<', '(')?.let { setOf(it) } ?: emptySet()
+        parseAuthorString(node["author"]?.textValue(), '<', '(')?.let { setOf(it) }.orEmpty()
```

Comment:
How about `.orEmpty().toSet()` to get rid of this `let` ?

Reply:
See above.